src/Dialect/Mlir/DialectBuilder.cpp

```diff
@@ -214,80 +214,101 @@ Value MathBuilder::constantIndex(int64_t val) const {
 }
 
 Value MathBuilder::negativeInf(Type type) const {
-  double value;
+  Value constant = nullptr;
```

Comment:
We cannot cast the int64_t minimum value to double. So we have to create `arith::ConstantOp` for each case instead of a single `arith::ConstantOp` with `double value`.